subscriptions/providers/apple_in_app/api.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+from typing import ClassVar
+
+import requests
+import tenacity
+from pydantic import (
+    BaseModel,
+    Field,
+    ValidationError,
+)
+
+from .enums import (
+    AppleEnvironment,
+    AppleValidationStatus,
+)
+from .exceptions import InvalidAppleReceiptError
+
+
+class AppleInApp(BaseModel):
+    # Several fields were omitted. For a full list go to
```

Comment:
Then maybe
```python
class Config:
     extra = ignore
```
?

Reply:
I believe that this is the default behaviour (tested it just a moment ago), but making it explicit will be much nicer. Thank you ^^

Reply:
Done.

Reply:
Ohhh, it's actually my bad, I thought it's `forbid` by default O_O